homeassistant/components/philips_js/switch.py

```diff
@@ -46,7 +48,7 @@ def __init__(
 
         super().__init__(coordinator)
 
-        self._attr_name = f"{coordinator.system['name']} Screen State"
+        self._attr_name = "Screen State"
```

Comment:
Sorry, just noticed this one. New styling naming is sentence-cased.

```suggestion
        self._attr_name = "Screen state"
```